### DISCUSSION TITLE
Set frame to start loop for Animation

### DISCUSSION THREAD

**Discussion by MyPingO** [godotengine/godot-proposals#7963] [Editor]
I thought it would be a neat feature to be able to loop an Animation and set what frame to loop from. 

Example: You have a falling animation for your character where the first 3 frames show the character animating his body to fall down, and the next 3 frames just move the character idly while falling, maybe some hair movement or something.

Godot already has the ability to set an animation to loop. I think it makes sense to be able to add a frame to start the loop from instead of making two animations and writing code to do this, it would be a nice quality-of-life change in the editor.

How to implement it?
Right-click on the animation timeline to set a loop point. Seems intuitive and straight forward.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/2159.
